.github/workflows/scala.yml

```diff
@@ -189,7 +189,7 @@ jobs:
 
       - name: Build JAR file
         run: |
-          cat /dev/null | sbt -Dsbt.log.noformat=true -J-Xmx2G -J-XX:+UseG1GC "set test in Test := {}" "set compile/skip := true" +package
+          cat /dev/null | sbt -Dsbt.log.noformat=true -J-Xmx3G -J-XX:+UseG1GC "set test in Test := {}" "set compile/skip := true" +package
```

Comment:
From https://github.com/cognitedata/cognite-sdk-scala/runs/6691511839?check_suite_focus=true#step:6:162 we failed at the step of publish package, should we add the same flag at line to increase the Ram size at line 198 as well ?😁

Reply:
Hmm, good question, I assumed it would be using the already created JAR file so it shouldn't need it, but I'll check the logs.

Reply:
Looks like it does for Scala 3 for some reason, but not (at least) for Scala 2.12. Might as well add that flag to that step just in case. Done, and good catch!

Reply:
cool cool hehe 🚀